presto-main/src/main/java/com/facebook/presto/operator/scalar/ReflectionParametricScalar.java

```diff
@@ -178,7 +178,7 @@ public ScalarFunctionImplementation specialize(BoundVariables boundVariables, in
         throw new PrestoException(FUNCTION_IMPLEMENTATION_MISSING, format("Unsupported type parameters (%s) for %s", boundVariables, getSignature()));
     }
 
-    public static List<SqlScalarFunction> parseDefinition(Class<?> clazz)
+    public static List<SqlScalarFunction> parseFunctionDefinitionClass(Class<?> clazz)
```

Comment:
I would just name this `parseFunctionDefinition` it's obvious from the argument that it parses a `Class`